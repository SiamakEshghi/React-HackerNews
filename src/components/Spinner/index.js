import React from 'react';
import ReactLoading from 'react-loading';

export default () => <ReactLoading color="#4A170C" type="spokes"/>