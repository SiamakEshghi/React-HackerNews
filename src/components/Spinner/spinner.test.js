import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Spinner from 'components/Spinner';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<Spinner />);
});

it('Spinner snapshot test', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
})