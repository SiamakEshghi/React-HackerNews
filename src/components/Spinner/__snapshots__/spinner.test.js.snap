// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner snapshot test 1`] = `
<Loading
  color="#4A170C"
  delay={0}
  height={64}
  type="spokes"
  width={64}
/>
`;
